Added localization support

angusm opened the pull request.

amacdonald-google committed:
Added localization support

Zoramite reviewed (commented):
Do you need to keep the _blueprint files in the example directory so that testing the example still works correctly?

Zoramite commented on kintaro/kintaro.py:
```diff
@@ -47,12 +48,114 @@ class UnknownReferenceError(Error):
+    localization_removed = field.split('@')[0]
+    if localization_removed[0] == '$':
+        return localization_removed[1:]
+    else:
```
You can remove this else and just return the variable at this point.

amacdonald-google committed:
Removed unnecessary else

amacdonald-google committed:
Adding blueprint files

angusm commented:
Hi Randy,

Good call on the blueprints, thank you :)
I've added them and removed the unnecessary else. Please let me know if there's anything else you'd like to see.
